ImpeccableHQ/swapr-dapp
# fix: add native to token

# Summary

Fixes #1713

This PR fixes a small bug where native "to token" wasn't being displayed. 

  # To Test

1. Click on the swap tokens button
2. Choose a native to token
- [ ] Verify that the to token displayed corresponds to the token selected / swapped
